spec/features/create_dataset_spec.rb

```diff
@@ -29,13 +30,19 @@
 
       expect(Dataset.where(title: "my test dataset").size).to eq(1)
 
-      # PAGE 2: Licence
+      # PAGE 2: Topic
+      choose option: topic.id
+      click_button "Save and continue"
+
+      expect(Dataset.last.topic.title).to eq(topic.title)
+
+      # PAGE 3: Licence
       choose option: "uk-ogl"
       click_button "Save and continue"
 
       expect(Dataset.last.licence).to eq("uk-ogl")
 
-      # Page 3: Location
+      # Page 4: Location
```

Comment:
Would removing the pointless page numbers be a better change here ;-)

Reply:
I'm honouring the design decisions of previous authors.

Reply:
OK. I don't think anyone would mind, though.